qemu/tests/set_link.py

```diff
@@ -154,6 +155,7 @@ def set_link_test(linkid, link_up, expect_status,
 
         """
         vm.set_link(linkid, up=link_up)
```

Comment:
this is kind of test case specific operations. If we don't check the network status right after the set_link then we don't need it. So I guess it is better to keep it in the test case. But still thanks for your advice.